### ISSUE TITLE
Generated Images Consistently Aligned to Top-Left Corner

### ISSUE THREAD

**Issue by yinlikestudy** [ximinng/LLM4SVG#24]
Hello,
I followed the tutorial and used Qwen2.5VL 7B as the base model. I conducted pre-training on the SVGXCore 250K dataset, followed by supervised fine-tuning (SFT) on the SVGX1M dataset, with a learning rate of 3e-4. During testing, I observed that the generated images consistently appear concentrated in the top-left corner of the output.

<img width="529" height="532" alt="Image" src="https://github.com/user-attachments/assets/ba137c2f-fb15-47cf-ae12-748dccceb449" />

Is this behavior expected, or could it indicate an issue with the training process or configuration? Have I potentially overlooked any critical steps or parameters during pre-training or fine-tuning? Any guidance would be greatly appreciated. Thank you!

**Comment by Qingman-Wu** [comment]
Hello, is your task text2svg? How many epochs did you train? How is the effect? I have 140,000 data, and the effect is still not ideal when epoch=10. Do you have any experience? Thank you very much.

**Comment by yinlikestudy** [comment]
Hello! I'm encountering an issue with both text2svg and img2svg. I trained on gen51k, vision25k, and un25k for 2 epochs. Have you come across this problem before?

**Comment by Qingman-Wu** [comment]
> Hello! I'm encountering an issue with both text2svg and img2svg. I trained on gen51k, vision25k, and un25k for 2 epochs. Have you come across this problem before?

I didn't encounter the problem that the generated images were all in the upper left corner, but my effect was not good. Could you please tell me how the effect you reproduce? This direction has troubled me for a long time. Thank you very much！！

**Comment by yinlikestudy** [comment]
> > Hello! I'm encountering an issue with both text2svg and img2svg. I trained on gen51k, vision25k, and un25k for 2 epochs. Have you come across this problem before?
> 
> I didn't encounter the problem that the generated images were all in the upper left corner, but my effect was not good. Could you please tell me how the effect you reproduce? This direction has troubled me for a long time. Thank you very much！！

Hello! I followed the approach in the paper, starting with pretraining and then performing supervised fine-tuning (SFT). Could you share which base model you used? Also, when you mentioned the 140k data points, are those from your own dataset?

**Comment by e-yi** [comment]
Hi @yinlikestudy , just to clarify — does this mean you followed the paper and implemented your own code for two-stage training, rather than using the one-stage approach suggested in the this repository?
